### PR TITLE
[DOCS] Add steps for fixing aliases and names in the CMS for term changes

### PR DESCRIPTION
**Pull request by swirtSJW**
## Description

Relates to this [Slack thread](https://dsva.slack.com/archives/C0FQSS30V/p1704910454485849) 

## QA steps

What needs to be checked to prove this works?
What needs to be checked to prove it didn't break any related things?
What variations of circumstances (users, actions, values) need to be checked?

As user facilities team member
   - [ ] Validate that the step added to the ACs in this template match the steps that [will one day be automated](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/7315) 

### Select Team for PR review

- [ ] `CMS Team`
- [ ] `Public websites`
- [x] `Facilities`
- [ ] `User support`
- [ ] `Accelerated Publishing`

### Is this PR blocked by another PR?

- [ ] `DO NOT MERGE`

### Does this PR need review from a Product Owner

- [ ] `Needs PO review`

### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
- [ ] Yes, and it's written in issue ____ and queued for publication.
  - [ ] Merge and ping the UX writer so they are ready to publish after deployment
- [ ] Yes, but it hasn't yet been written
  - [ ] Don't merge yet -- ping the UX writer to write and queue content
- [ ] No announcement is needed for this code change.
  - [ ] Merge & carry on unburdened by announcements

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/taxonomy-update-term.yml [hunk 1]
```diff
@@ -75,5 +75,9 @@ body:
       - [ ] Content is reviewed by relevant stakeholders
       - [ ] (List any necessary review and approval steps here)
       - [ ] Term is published
+      - [ ] Bulk resave system services to pick up name change (filtered for the old name)
```
Comment (swirtSJW) [inline .github/ISSUE_TEMPLATE/taxonomy-update-term.yml:78]:
```suggestion
      If term is a VA Service:
      - [ ] Bulk resave system services to pick up name change (filtered for the old name)
```

File: .github/ISSUE_TEMPLATE/taxonomy-update-term.yml [hunk 2]
```diff
@@ -75,5 +75,10 @@ body:
       - [ ] Content is reviewed by relevant stakeholders
       - [ ] (List any necessary review and approval steps here)
       - [ ] Term is published
+      If term is a VA Service:
```
Comment (davidmpickett) [inline .github/ISSUE_TEMPLATE/taxonomy-update-term.yml:78]: "If term is part of VA Service Taxonomy & the term name changes:"
Suggested for more specificity. This template is also used when updating an existing term without the term name changing so won't always apply."

Comment (swirtSJW) [inline .github/ISSUE_TEMPLATE/taxonomy-update-term.yml:78]:
```suggestion
      If term is part of VA Service Taxonomy & the term name changes:
```

**Review by davidmpickett — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/taxonomy-update-term.yml (hunk 2 above)
Comment (swirtSJW) [reply .github/ISSUE_TEMPLATE/taxonomy-update-term.yml]: Great suggestion.  I just update the PR.

**Pull request merged**